packages/client/machines/appMusicPlaylistsMachine.ts

```diff
@@ -94,11 +107,36 @@ export const appMusicPlaylistsModel = createModel(
     },
 );
 
+const stopPlaylistActor = appMusicPlaylistsModel.assign(
+    {
+        playlistsActorsRefs: ({ playlistsActorsRefs }, { roomID }) => {
+            const actor = playlistsActorsRefs.find(
+                (actor) => actor.id === roomID,
+            );
+
+            if (actor === undefined) {
+                console.error(
+                    'encountered unkown roomID in stop playlist actor',
+                );
+                return playlistsActorsRefs;
+            }
+
+            if (actor.ref.stop) {
+                actor.ref.stop();
+            }
```

Comment:
To stop an actor we should be using [`stop`](https://xstate.js.org/docs/guides/actors.html#stopping-actors) action.
As a consequence, we would have a first action to stop the actor, and another one to remove it from the context.

Actually, the less we interact with the actor imperatively inside the machine, the best it is.